continuous_integration/circle/build_doc.sh

```diff
@@ -16,8 +16,8 @@ conda update --yes --quiet conda
 conda create -n $CONDA_ENV_NAME --yes --quiet python=3
 source activate $CONDA_ENV_NAME
 
-conda install --yes --quiet pip numpy sphinx=1.6.3 matplotlib pillow
-pip install sphinx-gallery
+conda install --yes --quiet pip numpy sphinx=1.6.3 matplotlib pillow pandas
+pip install sphinx-gallery lz4
```

Comment:
Don't install lz4 here. IIUC this is only supposed to be installed when NUMPY_VERSION is  set.

Reply:
Not sure, this script is only used for the setting up the documentation: all potential dependencies are installed and this includes lz4 (see pandas, pillow, etc are also installed on the line just above)